package.json

```diff
@@ -24,10 +35,25 @@
     "sparkpost": "^2.1.2",
     "yargs": "^6.6.0"
   },
+  "devDependencies": {
+    "eslint": "^3.15.0",
```

Comment:
linting rules may in major versions. as we use default, i think we should change it to `~`.